R/pkg/tests/fulltests/data/test_utils_utf.json

```diff
@@ -0,0 +1,4 @@
+{"name": "안녕하세요"}
+{"name": "您好", "age": 30}
+{"name": "こんにちは", "age": 19}
+{"name": "Xin chào"}
```

Comment:
I cannot really figure why, but `\\N{LATIN SMALL LETTER A WITH GRAVE}` seems to mess up things on Windows.